tracer/src/Datadog.Trace/Tracer.cs

```diff
@@ -419,13 +419,25 @@ internal SpanContext CreateSpanContext(ISpanContext parent = null, string servic
 
                 if (traceContext == null)
                 {
-                    // If parent is SpanContext but its TraceContext is null, then it was extracted from
-                    // propagation headers. Create a new TraceContext (this will start a new trace) and initialize
+                    var propagatedTags = parentSpanContext.PropagatedTags;
+                    var samplingPriority = parentSpanContext.SamplingPriority;
+
+                    // When in appsec standalone mode, only distributed traces with the `_dd.p.appsec` tag
+                    // are propagated downstream, however we need 1 trace per minute sent to the backend, so
+                    // we unset sampling priority so the rate limiter decides.
+                    if (Settings?.AppsecStandaloneEnabledInternal == true)
+                    {
+                        // If the trace has appsec propagation tag, the default priority is user keep
+                        samplingPriority = propagatedTags?.GetTag(Tags.PropagatedAppSec) != "1" ? SamplingPriorityValues.Default : SamplingPriorityValues.UserKeep;
```

Comment:
Consider switching this logic to check for `== "1"` instead of `!= "1"`. I think it's easier to read 😅 


```suggestion
                        samplingPriority = propagatedTags?.GetTag(Tags.PropagatedAppSec) == "1" ? SamplingPriorityValues.UserKeep : SamplingPriorityValues.Default;
```

Reply:
More importantly, I think we want to set `samplingPriority = null` instead of `samplingPriority = SamplingPriorityValues.Default`. The value of `Default` is the same as `AutoKeep` (`1`):
https://github.com/DataDog/dd-trace-dotnet/blob/ea3597cec79f6324c316c39f09b976dd89db8fd8/tracer/src/Datadog.Trace/SamplingPriorityValues.cs#L20

By setting an actual value (instead of `null`), you will prevent any sampling mechanism to run on this trace (once a trace has a sampling priority, there's no need to make another decision).

Reply:
Oh Okay I see thanks, Yep I thought that the default was the "not set" value and that it will be changed in the process.
I'll set it to null then.

Reply:
Changed in c8c8c55